config/deploy.rb

```diff
@@ -55,7 +55,7 @@
 end
 
 before 'deploy:updated', 'loc:update_marc2bibframe2'
-before 'deploy:restart', 'shared_configs:pull'
+before 'shared_configs:update', 'deploy:restart'
```

Comment:
@shelleydoljack this is what I was trying to say. If you agree feel free to merge.

Reply:
I think it needs to be the other way around. See https://github.com/sul-dlss/mylibrary/blob/366ae4956344756ba8e756ac3f49d8aa76aa04a9/config/deploy.rb#L27-L28